chromium_src/chrome/browser/browser_about_handler.cc

```diff
@@ -13,6 +13,10 @@
 bool FixupBrowserAboutURLBraveImpl(GURL* url,
                                    content::BrowserContext* browser_context) {
   if (url->SchemeIs(kBraveUIScheme)) {
+    NOTREACHED() << "brave url should not be reached here. scheme mapping "
```

Comment:
why would we put a NOTREACHED() before a bunch of code? This does not seem correct to me